src/schema/v2/infiniteDiscovery/discoverArtworks.ts

```diff
@@ -39,14 +39,21 @@ export const DiscoverArtworks: GraphQLFieldConfig<void, ResolverContext> = {
       defaultValue: 2,
     },
   }),
-  resolve: async (_root, args, { artworksDiscoveryLoader }) => {
+  resolve: async (_root, args, { artworksDiscoveryLoader, meLoader }) => {
+    if (!meLoader || !artworksDiscoveryLoader) {
+      throw new Error("You need to be signed in to perform this action")
+    }
+
+    const me = await meLoader()
+
     const gravityArgs = {
       limit: args.limit,
       exclude_artwork_ids: args.excludeArtworkIds,
       mlt_fields: args.mltFields,
       liked_artwork_ids: args.likedArtworkIds,
       os_weights: args.osWeights,
       curated_picks_size: args.curatedPicksSize,
+      user_id: me.id,
```

Comment:
You can get user id from the context already, it's being provided by Force. You don't need `meLoader`